### DISCUSSION TITLE
[Request] - When is the next release?

### DISCUSSION THREAD

**Discussion by kei95** [excalidraw/excalidraw#8655] [General]
Hey team! I've been a fun of Excalidraw for quite some time. Since the last release was in Feb, I wonder if you guys have plan to release a new one soon. It's causing some troubles in packages depending on it as Excalidraw has grown quite a bit since then (e.g. https://github.com/excalidraw/excalidraw/discussions/new?category=general)

**Comment by dai-shi** [comment]
For context, @kei95 and I are working on [excalidraw-animate](https://github.com/dai-shi/excalidraw-animate) and it depends on [`@excalidraw/excalidraw`](https://www.npmjs.com/package/@excalidraw/excalidraw), which is not compatible with the latest one. @ad1992 What's the plan?

**Comment by Mrazator** [comment]
Hi, we didn't settle on a specific date, nor have we settled on all todos for the next release. Major change is switching to a new `esbuild` bundle, superseding the webpack one, hence it might take a bit longer.

> (e.g. [excalidraw/excalidraw/discussions/new?category=general](https://github.com/excalidraw/excalidraw/discussions/new?category=general))

Looks like an incorrect link.

**Reply by dai-shi** [reply]
I think @kei95 meant https://github.com/dai-shi/excalidraw-animate/pull/56.

@Mrazator Do you notice that the current version has an issue? It would be nice to fix it first before the major change.

**Reply by Mrazator** [reply]
I see, though we won't likely merge anything back to v17 - if that's what you are asking.

Instead, the first step might be releasing the next (RC) for you to take.

Also, I will make sure to start tracking steps for v18 release, once I have it, I will post it here.

**Comment by Mrazator** [comment]
Let's track it here. We shall hopefully release v18 by the end of this year.

https://github.com/excalidraw/excalidraw/issues/8771

**Comment by dai-shi** [comment]
@kei95 
https://x.com/excalidraw/status/1899510049057525981
